docker/ci/Dockerfile

```diff
@@ -20,6 +20,10 @@ RUN apk --no-cache add bash clang git g++ make nodejs openjdk8 python perl \
     && cd $(go env GOPATH)/src/github.com/GoASTScanner/gas \
     && git reset --hard d30c5cde3613e9ba0129febda849e4d4df1d57cd \
     && go install github.com/GoASTScanner/gas
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
```

Comment:
Yeah, we can use a specific nightly snapshot (e.g. `nightly-2017-02-25`). I could also try the beta channel instead of nightly, which might have all of the features we need.

Will start with using a specific nightly snapshot.